nsist/tests/test_local_wheels.py

```diff
@@ -0,0 +1,64 @@
+import unittest
+import os
+import platform
+import subprocess
+import glob
+
+from testpath.tempdir import TemporaryDirectory
+from testpath import assert_isfile, assert_isdir
+from nsist.pypi import fetch_pypi_wheels
+
+class TestLocalWheels(unittest.TestCase):
+    def test_matching_one_pattern(self):
+        with TemporaryDirectory() as td1:
+            subprocess.call(['pip', 'wheel', 'requests==2.19.1', '-w', td1])
```

Comment:
My bad, refactoring and forgot to reuse my variables. Fixed.

Reply:
Done. But it seems that the mechanism of launching `nosetests -a '!network'` does not work: no test (new or old) is executed.

I saw that nose is not maintained anymore, and it is advised to use another tests tool, like pytest in particular. Since it is quite the de facto tests framework for Python, would you like some tox + pytest in your plate sir ? ;)

Reply:
Weird, it works for me. I see (in master) 21 tests executed by `nosetests`, and 18 by `nosetests -a '!network'`.

Yes, I've been thinking about switching to pytest, which I already use for most other projects. Its [markers](https://docs.pytest.org/en/latest/example/markers.html) feature would work nicely for this. If you want to work on that, great! Let's do it as a separate PR, though.

I've never really got into using tox, though I understand its advantages. I may not use it myself, but I don't mind having a `tox.ini` file in the repo, so long as it's simple enough that it doesn't need much maintenance.

Reply:
Tox is really great for making independent python test environments. I use it a lot, it is completely straight forward to use it, no worries to have about maintenance.

Of course, in another PR anyway.